example-app/SantokuApp/src/components/overlay/loading/LoadingOverlay.tsx

```diff
@@ -1,19 +1,13 @@
 import React from 'react';
 import {ActivityIndicator} from 'react-native';
 
-import {FullWindowOverlay} from '../FullWindowOverlay';
 import {Overlay, OverlayProps} from './Overlay';
 
 const LoadingOverlay: React.FC<OverlayProps> = ({children, ...props}) => {
```

Comment:
Since `LoadingOverlay` (and `Snackbar`) no longer receive `children`, why don't you start using React.VFC?

Reply:
Fixed
c8e45e3d182990ec98e7edeb916227bf32bca060